CoreHistogramView/src/au/gov/asd/tac/constellation/views/histogram/DefaultBinCreatorProvider.java

```diff
@@ -32,164 +32,168 @@
  */
 @ServiceProvider(service = AttributeBinCreatorProvider.class)
 public class DefaultBinCreatorProvider extends AttributeBinCreatorProvider {
+    
+    private static final String BYTE = "byte";
```

Comment:
these should be `AttributeDescription` constants.

Reply:
You would think so, but it would appear that there aren't attribute descriptions for byte, double, or short

Reply:
orly... well that sounds like it needs a ticket :)